pkg/virt-config/deprecation/feature-gates.go

```diff
@@ -67,6 +69,8 @@ func init() {
 
 	RegisterFeatureGate(FeatureGate{Name: PasstGate, State: Discontinued, Message: PasstDiscontinueMessage, VmiSpecUsed: passtApiUsed})
 	RegisterFeatureGate(FeatureGate{Name: MacvtapGate, State: Discontinued, Message: MacvtapDiscontinueMessage, VmiSpecUsed: macvtapApiUsed})
+
+	RegisterFeatureGate(FeatureGate{Name: DisableCustomSELinuxPolicy, State: GA})
```

Comment:
I think `Discontinued` would be the correct `State`. @jean-edouard

Reply:
This PR effectively changes the default behavior of KubeVirt to match what that feature gate was doing.
Isn't that basically the definition of GA?

Reply:
Ah, you are right :)